JWPlayerPlugin/JWPlayerViewController.m

```diff
@@ -278,8 +278,8 @@ - (void)setPlayer:(JWPlayerController *)player {
     [player.view matchParent];
     
     self.player.fullscreen                 = NO;        // Fix for JP-1 task - hide fullscreen control
-    self.player.forceFullScreenOnLandscape = NO;
-    self.player.forceLandscapeOnFullScreen = NO;
+    self.player.forceFullScreenOnLandscape = YES;
```

Comment:
I would suggest at least doing it according to if the play inline or full screen function was the one called.

Reply:
Plus this was a requirement for the original implementation in team LATAM (@casajavi - FYI)
So the preferable solution is add a plugin configuration

Reply:
After testing again I noticed that I didn't need to change this booleans. I returned them both to NO